bartender/local_plugins/logger.py

```diff
@@ -21,7 +21,13 @@ def __getattr__(self, attr):
 
 
 # Since we are imitating the logging module, we will allow camel case method names
-def getPluginLogger(name, format_string=None, log_directory=None, log_name=None):
+def getPluginLogger(
+        name,
+        format_string=None,
+        log_directory=None,
+        log_name=None,
+        log_level=None,
+):
```

Comment:
Is this how we should be formatting things now?

Reply:
The reason I do this now, is because at PyCon I found out that this is easier for blind users to read. As it turns out, it doesn't look half bad either, because it keeps vertical alignment.